src/cbmc/cbmc_parse_options.cpp

```diff
@@ -1153,6 +1153,8 @@ void cbmc_parse_optionst::help()
     " --classpath dir/jar          set the classpath\n"
     " --main-class class-name      set the name of the main class\n"
     // NOLINTNEXTLINE(whitespace/line_length)
+    "--java-max-recursion-depth    max recursion depth for nondet init of recursive objects\n"
```

Comment:
While I do accept that the initialisation code currently exists for Java only: I don't think this command-line option should remain as specific. Furthermore the option name suggests that functional recursion is limited, whereas this is all about object initialisation. How about: "recursive-object-bound"?